scripts/pre_commit_linter.py

```diff
@@ -959,6 +959,9 @@ def __init__(self, all_filepaths, verbose_mode_enabled=False):
             verbose_mode_enabled: bool. True if verbose mode is enabled.
         """
         # Set path for node.
+        # The path for node is set since the lint test fail on circleci
```

Comment:
Wording nit: 

... is set explicitly, since otherwise the lint tests fail on CircleCI due to the TypeScript files not being compilable.

Reply:
Done.